libqtile/widget/memory.py

```diff
@@ -19,25 +18,25 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 from __future__ import division
+
+import psutil
+
 from libqtile.widget import base
 
 
 def get_meminfo():
     val = {}
-    with open('/proc/meminfo') as file:
-        for line in file:
-            key, tail = line.split(':')
-            uv = tail.split()
-            val[key] = int(uv[0]) // 1000
-    val['MemUsed'] = val['MemTotal'] - val['MemFree']
+    val['MemUsed'] = int(psutil.virtual_memory().used / 1024 / 1024)
+    val['MemTotal'] = int(psutil.virtual_memory().total / 1024 / 1024)
+    val['MemFree'] = val['MemTotal'] - val['MemUsed']
```

Comment:
There's a potential config breakage for people doing stuff [like this](https://github.com/qtile/qtile/pull/1216#issuecomment-438977640) here.

Maybe we could accept that config breakage but then expose all the stuff contained in `psutil.virtual_memory()`, so that people actually can replace what they used to do? We could then drop the useless mapping in names.

For the same reason, I would also add `psutil.swap_memory()` informations here and not in a separate widget (plus I don't feel like the two widgets will benefit from being separated, the code being identical).

Reply:
Good point. It would be easy to merge that into memory.py. I basically did the same on graph.py. I'll get on that.